content/file/example_test.go

```diff
@@ -79,20 +78,18 @@ func Example_packFiles() {
 	// 1. Add files into the file store
 	mediaType := "example/file"
 	fileNames := []string{"foo.txt", "bar.txt"}
-	fileDescriptors := make([]ocispec.Descriptor, 0, len(fileNames))
 	for _, name := range fileNames {
 		fileDescriptor, err := store.Add(ctx, name, mediaType, "")
 		if err != nil {
 			panic(err)
 		}
-		fileDescriptors = append(fileDescriptors, fileDescriptor)
```

Comment:
We still need this as layers to pass to oras.PackManifestOptions{}

Reply:
Thanks for catching!